.github/workflows/spread-tests.yaml

```diff
@@ -98,6 +103,16 @@ jobs:
               fi
           fi
 
+    - name: Populate systems
```

Comment:
you don't need to do that if you need to run in all the supported systems.

you can run `spread <backend>:<task>` which will run in all the supported systems

Reply:
Listing out systems seemed like the least-invasive solution. I'll need to rewrite some steps to allow for what you say and add some extra logic to determine the name of the spread logs to upload. I would need to add in some potentially confusing logic to maintain the current behavior that passing an empty string as systems means to execute on no systems (as with ubuntu-no-lts). I introduced the `*` to distinguish the case of all systems instead of none.

Reply:
Also, we do have a check on line 233 to see if the systems include amazon-linux. To make sure that logic doesn't fall through the cracks in future workflow calls, I would need to call `spread -list` there to ensure one of the systems isn't that one.